npcs/carolina_jones.md

```diff
@@ -0,0 +1,12 @@
+# Carolina Tomes
```

Comment:
Why did you get rid of the Previous Missions section?

Reply:
This was something I lost trying to bring the branch back up to speed with the file reorganization. I added it back in.